### DISCUSSION TITLE
I'm looking for an example of how to run a blazor .net 8 project in photino.NET

### DISCUSSION THREAD

**Discussion by RaySinner** [tryphotino/photino.NET#206] [Q&A]
I'm looking for an example of how to run a blazor .net 8 project in photino.NET. I know that there is a separate project called photino blazor, but it would be useful for me to have an implementation exactly like photino.NET

**Comment by Xyncgas** [comment]
Clone ``` https://github.com/tryphotino/photino.Blazor.git ```
As of today it's bugged on some platform see https://github.com/tryphotino/photino.Blazor/issues/138
now simply run ``` dotnet run ``` inside the directory ``` /Samples/Photino.Blazor.Sample ```

PS : I see you already mentioned photino.blazor, but it's not a separate project it is using photino.net
Are you implementing blazor scenario without using photino.blazor that's available wow that's courageous !

**Comment by MikeYeager** [comment]
Closing unless you have further questions.
